File: extensions/shared/src/main/java/app/revanced/extension/youtube/patches/HideSeekbarPatch.java
package app.revanced.extension.youtube.patches;

import app.revanced.extension.youtube.settings.Settings;

@SuppressWarnings("unused")
public class HideSeekbarPatch {
    public static boolean hideSeekbar() {
        return Settings.HIDE_SEEKBAR.get();
    }
}

File: patches/src/main/kotlin/app/revanced/patches/youtube/layout/hide/seekbar/HideSeekbarPatch.kt
package app.revanced.patches.youtube.layout.hide.seekbar

import app.revanced.patcher.patch.bytecodePatch
import app.revanced.patches.youtube.interaction.seekbar.hideSeekbarPatch

@Deprecated("Patch was moved to app.revanced.patches.youtube.interaction.seekbar")
val hideSeekbarPatch = bytecodePatch {
    dependsOn(
        hideSeekbarPatch
    )
}
